src/main/java/ch/puzzle/eft/controller/ValidationController.java

```diff
@@ -0,0 +1,19 @@
+package ch.puzzle.eft.controller;
+
+import ch.puzzle.eft.model.ExamNumberForm;
+import jakarta.validation.Valid;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class ValidationController {
```

Comment:
Der ValidationController kann ebenso gut in den SiteController integriert werden. Damit haben wir alle wichtigen Requests in demselben Controller.